File: app/src/main/java/com/numerad/quickseries/json/Gps.java
package com.numerad.quickseries.json;

import com.google.gson.annotations.SerializedName;

import javax.annotation.Generated;

@Generated("com.robohorse.robopojogenerator")
public class Gps{

	@SerializedName("latitude")
	private String latitude;

	@SerializedName("longitude")
	private String longitude;

	public void setLatitude(String latitude){
		this.latitude = latitude;
	}

	public String getLatitude(){
		return latitude;
	}

	public void setLongitude(String longitude){
		this.longitude = longitude;
	}

	public String getLongitude(){
		return longitude;
	}

	@Override
 	public String toString(){
		return 
			"Gps{" + 
			"latitude = '" + latitude + '\'' + 
			",longitude = '" + longitude + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/ztd/interview_test/infrustructure/data/models/restaurant/Gps.kt
package com.ztd.interview_test.infrustructure.data.models.restaurant

import com.google.gson.annotations.SerializedName

data class Gps(

	@SerializedName("latitude")
	val latitude: String? = null,

	@SerializedName("longitude")
	val longitude: String? = null
)